root/release_group/merge.tt

```diff
@@ -14,11 +14,14 @@
                     <th>[%- l('Type') -%]</th>
                     <th class="year c">[%- l('Year') -%]</th>
                     <th class="count c">[%- l('Releases') -%]</th>
+                    [% IF to_merge.size > 2 %]
+                        <th style="width: 1em" />
```

Comment:
Can we add `aria-label="[% l('Remove from merge') %]"` to the th here since it has no label otherwise?